### ISSUE TITLE
[T3][P1] Test de primalidad

### ISSUE THREAD

**Issue by DiegoEmilio01** [marceloarenassaavedra/IIC2283-2-21#57]
¿El test de primalidad utilizado debe ser obligatoriamente el de Solovay-Strassen? Esto lo pregunto porque la nota al pie de página es utilizada en la descripción del problema 1, no en las hints como si fuese esta una sugerencia. Muchas gracias de antemano por la respuesta.

**Comment by marceloarenassaavedra** [comment]
Sí, se debe utilizar la versión del test de Solovay-Strassen visto en clases.

Saludos!
